### ISSUE TITLE
Package `tla-ts-mode` is unavailable

### ISSUE THREAD

**Issue by philipmw** [Davidbrcz/tla-ts-mode#2]
The readme says to use `(use-package ...)` to install this package. Yet this package is not available in MELPA nor MELPA Stable, so `use-package` fails:

> Error (use-package): Failed to install tla-ts-mode: Package ‘tla-ts-mode’ is unavailable

**Comment by Davidbrcz** [comment]
Yes, I know. I intend to publish it on MELPA at some point.  But I'm not really happy with the state of the features for moment.

A workaround is to use straight + use-package, or clone it directly and use [use-package :load-path](https://github.com/jwiegley/use-package#extending-the-load-path)

So in the mean time, I'll leave this issue open.
